coldfront/plugins/qumulo/hooks.py

```diff
@@ -0,0 +1,63 @@
+import textwrap
+
+from coldfront.config.env import ENV
+from coldfront.core.utils.mail import EMAIL_ENABLED, send_email
+
+def acl_reset_complete_hook(task_object):
+    task_duration = '{:.2f}'.format(task_object.time_taken())
```

Comment:
@ris-bmulligan I would refactor out into at least two methods:

1. calculate what goes into the variables
2. another handling the send email or write to file responsibilties.